core/crypto/parser.go

```diff
@@ -23,7 +23,8 @@ import (
 const (
 	chars      = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
 	numbers    = "0123456789"
-	everything = chars + numbers + "~!@#$%^&*()-_+={}[]\\|<,>.?/\"';:`"
+	symbols    = "~!@#$%^&*()-_+={}[]\\|<,>.?/\"';:`"
```

Comment:
no double quotes

Reply:
I think we can merge it w/o double quotes, but it looks like we should made these configurable later down the line.

I am sure different people will have different password policies, what characters to allow/deny etc.

Reply:
Changed symbols to only password support special chars.